nncf/quantization/algo.py

```diff
@@ -112,10 +112,10 @@ class QuantizerGroup(Enum):
 
 class NonWeightQuantizerInfo:
     def __init__(self, quantizer_module_ref: BaseQuantizer,
-                 affected_ia_op_exec_contexts: List[InputAgnosticOperationExecutionContext],
+                 affected_insertions: List[QuantizerInsertionInfo],
                  quantizers_between_quantizable_layers: QuantizersBetweenQuantizableLayers = None):
         self.quantizer_module_ref = quantizer_module_ref
-        self.affected_ia_op_exec_contexts = affected_ia_op_exec_contexts
+        self.affected_insertions = affected_insertions
```

Comment:
The `InputAgnosticOperationExecutionContext` is an address of a *node*, but the insertion of a hook for a node can now proceed in two ways - either post- or pre-. Therefore a position of a quantizer is no longer defined by a node, but by a node and the insertion type.